hw/ip/sysrst_ctrl/doc/_index.md

```diff
@@ -101,7 +104,37 @@ Software can program the `sysrst_ctrl` block to detect edge transitions on the `
 For example, when the power button is pressed, `pwrb_in` goes from logic 1 to logic 0 which would amount to an H->L transition.
 Likewise, when the power button is released, `pwrb_in` goes from logic 0 to logic 1 which would amount to an L->H transition.
 When `sysrst_ctrl` detects a transition (H->L or L->H) as specified in {{< regref KEY_INTR_CTL >}} and it meets the debounce requirement in {{< regref KEY_INTR_DEBOUNCE_CTL >}}, `sysrst_ctrl` sets the {{< regref KEY_INTR_STATUS >}} register to indicate the interrupt cause and send out a consolidated interrupt to the PLIC.
-The software interrupt handler should then read {{< regref KEY_INTR_STATUS >}} register and clear the interrupt.
+The software interrupt handler should then read the {{< regref KEY_INTR_STATUS >}} register and clear the interrupt via the {{< regref INTR_STATE >}} register.
+
+Note that an interrupt will also issue a wakeup request to the OpenTitan power manager via `aon_ot_wkup_req_o`.
+Software should therefore read and clear the {{< regref WKUP_STATUS >}} register as well.
+
+## Ultra-low-power Wakeup Feature
+
+Software can program the `sysrst_ctrl` block to detect certain specific signal transitions on the (possibly inverted) `ac_present`, `pwrb_in` and `lid_open` inputs.
+As opposed to the combo detection and general key interrupt features above, this is a fixed function feature with limited configurability. 
+In particular, the transitions that can be detected are fixed to the following:
+
+- A high level on the (potentially inverted) `ac_present` signal
+- A H -> L transition on the (potentially inverted) `pwrb_in` signal
+- A L -> H transition on the (potentially inverted) `lid_open` signal
```

Comment:
@toshaq can you take a look at this description and let me know whether this is correct?